UnitySDK/Assets/ML-Agents/Scripts/DemonstrationRecorder.cs

```diff
@@ -15,24 +15,37 @@ public class DemonstrationRecorder : MonoBehaviour
         private string filePath;
         private DemonstrationStore demoStore;
 
-        /// <summary>
-        /// Initializes Demonstration store.
-        /// </summary>
```

Comment:
Can we replace comment rather than delete it ?

Reply:
The comment was moved, not deleted.